great-idea/css/index.css

```diff
@@ -63,4 +63,271 @@ h1, h2, h3, h4, h5 {
+}
+
+.cta .cta-text button {
+	background: white;
+	vertical-align: top;
+	margin-top: 6px;
+	padding: 8px 44px;
+	font-size: 15px;
+	border: 1px solid black;
+	cursor: pointer;
+	animation: slidein 2s ease-in 1s both running;
+}
+
+@keyframes slidein {
+	from {
+	  margin-left: 600%;
+	  width: 150%; 
+	}
+	  
+	to {
+	  margin-left: 0%;
+	  width: 55%;
+	}
+}
+
+.cta .cta-text button:hover {
+	background: black;
+	color: white;
+}
+
+.cta img {
+	display: inline-block;
+	margin-top: 69px;
+}
+
+.main-content {
+	border-top: 1px solid black;
+	margin-top: 76px;
+	padding-top: 39px;
+}
+
+.main-content .top-content {
+	padding: 0 10px;
+	display: flex;
+}
+
+.main-content .top-content .text-content {
+	width: 410px;
+	margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+	margin-right: 0;
+}
+
+.main-content .middle-img {
+  	margin-top: 45px;
+}
+
+.main-content .bottom-content {
+	margin-top: 37px;
+	padding: 0 10px;
+	display: flex;
+}
+
+.main-content .bottom-content .text-content {
+	width: 275px;
+	margin-right: 12px;
+}
+
+.main-content .bottom-content .text-content:last-child {
+  	margin-right: 0;
+}
+
+.contact {
+	margin-top: 57px;
+	border-top: 1px solid black;
+	padding: 39px 10px 0;
+}
+
+.contact p {
+  	margin: 20px 0;
+}
+
+footer {
+	margin: 32px 0;
+	text-align: center;
+	font-size: 14px;
+}
+
+.toppage {
+	padding-bottom: 30px;
+	border-bottom: solid 2px black;
+}
+
+.toppage h1 {
+	font-size: 30px;
+}
+
+.midpage{
+	display: flex;
+	flex-flow: wrap;
+	justify-content: space-between;
+	padding: 20px 0;
+	border-bottom: 2px solid black;
+	box-sizing: content-box;
+}
+.midpage button{
+	padding: 5px;
+	border-radius: 8px;
+	background: white;
+	margin: 10px;
+}
+
+.midpage button:hover{
+	background: gray;
+	color: white;
+}
+.midpage div {
+	margin: 20px 0;
+	width: 400px;
+	padding: 10px;
+	border: 1px solid black;
+	background: lightgray;
+}
+
+.bottompage {
+	padding: 30px 0;
+	display: flex;
+}
+
+.bottompage p {
+	padding: 20px 40px 20px 0;
+}
+
+ul {
+	margin-left: 20px;
+	list-style-type: square; 
+}
+
+li {
+	padding: 4px 0;
+}
+
+.actualcontacts section {
+	display: flex;
+	flex-wrap: wrap;
+	padding: 20px 0 20px 20px;
+	border: 1px solid black;
+	background: lightgray;
+}
+
+.actualcontacts h1{
+	border-bottom: 2px solid black;
+	padding: 20px 0;
+	text-align: center;
+	font-size: 30px;
+}
+
+.actualcontacts div {
+	width: 400px;
+	margin: 10px;
+	border: 1px double black;
+	border-radius: 8px;
+}
+
+.actualcontacts img{
+	border-radius: 12px;
+}
+
+.text {
+	padding: 70px 0;
+	text-align: center;
+}
+
+.text button{
+	margin: 25px;
+	position: relative;
+	background-color: gray;
+	font-size: 15px;
+	padding: 10px;
+	text-align: center;
+	text-decoration: none;
+	overflow: hidden;
+	cursor: pointer;
+}
+
+.text button:after {
+	content: "";
+	background: #f1f1f1;
+	display: block;
+	position: absolute;
+	padding-top: 300%;
+	padding-left: 350%;
+	margin-left: -20px !important;
```

Comment:
I love the animation.  I tried to play with it a bit.  The :after pseudo class eludes me on how you used it.  The only thing I had to comment would be that generally buttons lead to a different page, so perhaps an animation on hover would be better for a button?